include/Fader.h

```diff
@@ -145,10 +169,16 @@ class LMMS_EXPORT Fader : public QWidget, public FloatModelView
 
 	QPixmap m_knob {embed::getIconPixmap("fader_knob")};
 
+	// Stores the offset to the knob center when the user drags the fader knob.
+	// This is needed to make it feel like the users drag the knob without it
+	// jumping immediately to the click position.
+	int m_knobCenterOffset {0};
+
 	bool m_levelsDisplayedInDBFS {true};
+	bool m_modelIsLinear {false};
 
-	int m_moveStartPoint {-1};
-	float m_startValue {0.};
+	// The dbFS amount after which we drop down to -inf dbFS
```

Comment:
We could consider moving this into the `public` section of the `Fader` class as a `static constexpr` variable as one could say this is part of the public API. Something like `static constexpr auto MinimumDbValue = -120.0f`.

```suggestion
	//! The dbFS amount after which we drop down to -inf dbFS
```

Reply:
I have turned this into a `constexpr` in the implementation file with commit ccae1b5daa1.

IMO it should not go into the public section as it is an implementation detail of the `Fader` that currently cannot be changed by any client anyway. If a client ever becomes interested in that minimum value then the code can be adjusted into that direction. I would consider doing it now "[YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) territory".